### PR TITLE
[Mailer] add more info for debugging

### PR DESCRIPTION
**Pull request by 94noni**
Hi

I did not understand this section of getDebug until I navigate the code and see the swap to do with
- MailerInterface and TransportInterface

`MailerInterface`

$a = $this->mailer->send($email); // $a is null

`TransportInterface`

$a = $this->mailer->send($email); // $a is SentMessage

### CODE REVIEW

File: mailer.rst [hunk 1]
```diff
@@ -641,6 +641,12 @@ provides access to the original message (``getOriginalMessage()``) and to some
 debug information (``getDebug()``) such as the HTTP calls done by the HTTP
 transports, which is useful to debug errors.
 
+.. note::
+
+    You will need to replace the typehint :class:`Symfony\\Component\\Mailer\\MailerInterface`
```
Comment (OskarStark) [inline mailer.rst]:
```suggestion
    You will need to replace :class:`Symfony\\Component\\Mailer\\MailerInterface`
```

File: mailer.rst [hunk 2]
```diff
@@ -641,6 +641,12 @@ provides access to the original message (``getOriginalMessage()``) and to some
...
+    :class:`Symfony\\Component\\Mailer\\Transport\\TransportInterface` to have
```
Comment (OskarStark) [inline mailer.rst]:
```suggestion
    with :class:`Symfony\\Component\\Mailer\\Transport\\TransportInterface` to have
```

**Comment by OskarStark** [comment]
Open to finish this PR?

**Comment by 94noni** [comment]
@OskarStark thx for reviewing, here you are PR updated :)

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
This is already the case in 5.4 branch, isn't it?

**Comment by javiereguiluz** [comment]
Thanks Antoine! We merged this in 5.4 branch and up.

**Pull request merged**